storage/src/vespa/storage/frameworkimpl/status/statuswebserver.h

```diff
@@ -72,14 +76,11 @@ class StatusWebServer : private config::IFetcherCallback<vespa::config::content:
                     framework::StatusReporterMap&,
                     const config::ConfigUri & configUri);
     ~StatusWebServer() override;
-
-    void handlePage(const framework::HttpUrlPath&, std::ostream& out);
-    static vespalib::string getServerSpec(const vespalib::string &requestSpec,
-                                          const vespalib::string &serverSpec);
+    int getListenPort() const;
+    void handlePage(const framework::HttpUrlPath&, vespalib::Portal::GetRequest request);
 private:
     void configure(std::unique_ptr<vespa::config::content::core::StorStatusConfig> config) override;
-    void getPage(const char* url, Fast_HTTPConnection& conn);
-    void run(framework::ThreadHandle&) override;
+    void run(framework::ThreadHandle&) override {}
```

Comment:
It looks safe to remove the inheritance from `Runnable` entirely, but that can be done as a follow-up.

Reply:
will do. I missed the separate inheritance when just chopping away code earlier.